unimol/pyproject.toml

```diff
@@ -0,0 +1,58 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "unimol"
```

Comment:
should we change the package names to unimol-albert or something?

Reply:
I thought about this -- what I wasn't sure on was if that means we have to go through and edit all the sources where we are importing it becuase I would prefer to not have to do that -- if changing the package name doesn't change the installed package name then we can publish it under a different name sure

Reply:
Yeah we would have to change the downstream users of thsi to install the new package name -- let's leave it as is for now